server/lib/sync-agent.js

```diff
@@ -314,7 +314,14 @@ export default class SyncAgent {
       import_start_date: moment().subtract(this.ship.private_settings.import_days, "days").format()
     };
 
-    const wrappedQuery = this.adapter.in.wrapQuery(query, replacements);
+    let wrappedQuery;
+    try {
+      query = removeComments(query, replacements, this.hull);
+      wrappedQuery = this.adapter.in.wrapQuery(query, replacements);
```

Comment:
Second call to function here. Wrapped in try catch otherwise the error isn't shown in the preview editor.